python-package/xgboost/testing/__init__.py

```diff
@@ -668,8 +668,7 @@ def weight_margin(draw: Callable) -> TestDataset:
 
 # A strategy for drawing from a set of example datasets. May add random weights to the
 # dataset
-@memory.cache
```

Comment:
Removed since the composite strategy cannot be pickled. joblib has been issuing warnings for a while.